nautobot/extras/health_checks.py

```diff
@@ -26,3 +28,34 @@ def check_status(self):
             raise ServiceReturnedUnexpectedResult("Integrity Error")
         except DatabaseError:
             raise ServiceUnavailable("Database error")
+
+
+class RedisBackend(BaseHealthCheckBackend):
+    """Health check for Redis."""
+
+    redis_url = getattr(settings, "CACHEOPS_REDIS", "redis://localhost:6379/1")
+    sentinel_url = getattr(settings, "CACHEOPS_SENTINEL", None)
+
+    def check_status(self):
+        """Check Redis service by pinging the redis instance with a redis connection."""
+        # if CACHEOPS_REDIS is set to False or Sentinel is enabled
+        # We need to check Redis using Sentinel
+        if not self.redis_url or self.sentinel_url is not None:
```

Comment:
The below logic expects that `sentinel_url` must have a dictionary value or it will throw an exception - plus I don't think it makes sense in general to try Sentinel at all if `sentinel_url` is not set?

```suggestion
        if self.sentinel_url is not None:
```